expr/http_service.go

```diff
@@ -174,13 +203,38 @@ func (svc *HTTPServiceExpr) EvalName() string {
 	return fmt.Sprintf("service %#v", svc.Name())
 }
 
-// Prepare initializes the error responses.
+// Prepare initializes the error responses and webhooks.
 func (svc *HTTPServiceExpr) Prepare() {
 	// Create routes for JSON-RPC endpoints if needed
 	if svc.ServiceExpr.Meta != nil && svc.ServiceExpr.Meta["jsonrpc:service"] != nil {
 		svc.prepareJSONRPCRoutes()
 	}
 
+	// Prepare webhooks - map service webhooks to HTTP webhooks
+	for _, w := range svc.ServiceExpr.Webhooks {
```

Comment:
I wonder if we really need transport agnostic webhooks? seems pretty HTTP specific by definition. Maybe we just have HTTPWebhooks (which we could just call Webhooks)